### PR TITLE
Change to path handling for ab-test; minor edits to markdown-to-html 

### PR DESCRIPTION
**Pull request by ecarr-llnw**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**/
+typing_extensions.py
+zipp.py
```
Comment (srigarlapati) [inline .gitignore:4]: I think it's probably best to delete this file as to not include your personal .gitignore in the edge-functions repo.

File: sample-functions/python3/ab-test/index.py [hunk 2]
```diff
@@ -46,7 +46,9 @@ def handler(request, context):
 
         filtered = list(filter(lambda host: host['weight'] > random, hosts))
 
-        redirect = filtered[0]['name'] + "/" + path
+        if path:
+            path = "/" + path
+        redirect = filtered[0]['name'] + path
```
Comment (srigarlapati) [inline sample-functions/python3/ab-test/index.py:51]: With this change you would also need to recompress this file to create an updated index.zip file.

File: sample-functions/python3/markdown-to-html/index.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    return {
+        "statusCode": 200,
+        "body": result,
+        "headers": {
+            "Content-Type": type,
+            "X-LLNW-Faas-Collect-Stdio": "1"
+        }
+    }
+
+# def test():
```
Comment (srigarlapati) [inline sample-functions/python3/markdown-to-html/index.py]: Remove commented out code?

**Review by srigarlapati — CHANGES_REQUESTED** [review]

File: sample-functions/python3/markdown-to-html/index.py (hunk 3 above)
Comment (ecarr-llnw) [reply sample-functions/python3/markdown-to-html/index.py]: done

File: .gitignore (hunk 1 above)
Comment (ecarr-llnw) [reply .gitignore:4]: Can this file just be deleted from the pull request? If not I guess I need to close this PR and do another one.

File: sample-functions/python3/ab-test/index.py (hunk 2 above)
Comment (ecarr-llnw) [reply sample-functions/python3/ab-test/index.py:51]: Holding off on zip pushes pending clarification from the team.

File: .gitignore (hunk 1 above)
Comment (srigarlapati) [reply .gitignore:4]: This file can be deleted from the PR.

Just need to delete the file then push up your changes

Comment (colinbarr) [reply .gitignore:4]: You can add a commit to remove it into your fork, yes. You can also rebase/squash your commits to make a single commit with all your changes if you'd prefer.

**Pull request closed without merging**